add topic naming note (#939)
apache/pulsar-site: docs/admin-api-topics.md

non-persistent://tenant/namespace/topic
```

:::note

> Topic naming: due to backward compability, some special characters, eg., "/", is allowed as a part of topic name. But we recommended that you should avoid using special characters as a part of topic name.

Reviewer: ```suggestion
Topic naming: due to backward compability, some special characters, eg., "/", is allowed as a part of topic name. But we recommended that you should avoid using special characters as a part of topic name.

:::
```

non-persistent://tenant/namespace/topic
```

:::note

> Topic naming: due to backward compability, some special characters, eg., "/", is allowed as a part of topic name. But we recommended that you should avoid using special characters as a part of topic name.

Reviewer: ```suggestion
Topic naming: due to backward compatibility, some special characters, e.g., "/," are allowed as a part of the topic name. But it's recommended not to use special characters as a part of the topic name.
```
